spec/requests/custom_reactions_controller_spec.rb

```diff
@@ -155,6 +155,21 @@
     end
   end
 
+  context 'changing discourse_reactions_like_icon' do
```

Comment:
are we changing the icon or the reaction? Be very careful with wording please

Reply:
oooh my bad we are changing `reaction`